Makefile

```diff
@@ -41,7 +43,8 @@ build:
 	@go build ${GO_CMD_FLAGS} -o ${CONTROLLER_BOOTSTRAP} ./cmd/controller-bootstrap/main.go
 
 generate: build
-	@${CONTROLLER_BOOTSTRAP} generate -s ${AWS_SERVICE} -r ${ACK_RUNTIME_VERSION} -v ${AWS_SDK_GO_VERSION} -d=${DRY_RUN} -e=${EXISTING_CONTROLLER} -o ${ROOT_DIR}/../${AWS_SERVICE}-controller -m ${SERVICE_MODEL_NAME}
+	@${CONTROLLER_BOOTSTRAP} generate -s ${AWS_SERVICE} -r ${ACK_RUNTIME_VERSION} -v ${AWS_SDK_GO_VERSION} -d=${DRY_RUN} \
+    -e=${EXISTING_CONTROLLER} -o ${ROOT_DIR}/../${AWS_SERVICE}-controller -m ${SERVICE_MODEL_NAME} -c ${TEST_INFRA_HEAD_COMMIT_SHA}
```

Comment:
Given how many flags there are now, I might suggest that we use the long form for everything (`--model-name`, `--service-name`, etc.). I think short codes are great when you're doing stuff quickly on the command line, but in scripts we want the full transparency around what is what

Reply:
Thank you for the suggestion, the flags have been modified to use long form.